### ISSUE TITLE
Colons in nested f-string expressions cause invalid "unterminated" error

### ISSUE THREAD

**Issue by amachanic** [microsoft/pylance-release#206]
## Environment data
-   Language Server version: 2020.8.0
-   OS and version: Centos 7
-   Python version (& distribution if applicable, e.g. Anaconda): 3.6.3

## Expected behaviour

No issue

## Actual behaviour

![image](https://user-images.githubusercontent.com/39809964/89541174-02e3b880-d7cc-11ea-8aa3-7b47049281ed.png)

## Code Snippet / Additional information

```python
x = f"""
    {
        f'''
            {':'}
        '''
    }
"""
```

Also: Huge thanks to @jakebailey and @erictraut for the quick response on my prior issue! Keep up the good work!

**Comment by jakebailey** [comment]
I believe we tested for all of these cases in MPLS, might be a good idea to open those test files to make sure we aren't missing any more past this one.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright & Pylance.

@jakebailey, good suggestion about looking at the MPLS test cases. They didn't include this particular form of f-string nesting with a colon, but they covered a lot of other cases.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.8.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202081-13-august-2020
